src/docs/PCA/PCAUserGuide.adoc

```diff
@@ -35,23 +35,32 @@ include::references/HDFS-SPARK.adoc[]
 
 include::references/Storm.adoc[]
 
-=== Zookeeper
+=== Kafka
 
-image::PCA_zookeeper.png[align=center]
+The Kafka service allows to deploy through ProActive Cloud Automation (PCA) Portal a Kafka broker. 
 
-=== Kafka
+image::PCA_kafka.png[align=center][100000,2000]
 
-image::PCA_kafka.png[align=center]
+The instance will be in the state running after the deployment.
```

Comment:
It could be good to explain the behaviour of the Kafka instance if the Zookeeper is stopped.